projects/asthma-copd/README.md

```diff
@@ -0,0 +1,76 @@
+# PhenoPLIER analyses of Asthma-COPD Overlap Syndrome (ACOS)
```

Comment:
Do you want to fill this file out in more detail, or will you do that in a future PR?

Reply:
Yes, I will fill it out in the future, it doesn't make much sense to do that now. I've updated it though with some minor changes.